src/ngraph/runtime/cpu/mkldnn_emitter.cpp

```diff
@@ -36,6 +36,7 @@ MKLDNNEmitter::~MKLDNNEmitter()
 {
     for (auto p : m_mkldnn_primitives)
         delete p;
+    mkldnn_utils::mkl_serv_free_buffers();
```

Comment:
@gaurides Can you please add a description , explaining  the role of `serv_free_buffers` ?